evalir.ml

```diff
@@ -423,11 +423,11 @@ struct
       unblock out2;
       apply_cont cont env (`Record [])
     (* end of session stuff *)
-    | `PrimitiveFunction ("unsafeAddRoute", _), [pathv; handler; error_handler] ->
```

Comment:
This looks like it undoes a change that was made to Links since you created your branch, we should be careful not to undo such changes.